### ISSUE TITLE
Procedure to enable all of the reports

### ISSUE THREAD

**Issue by gmaxey** [electric-cloud-community/EC-AuditReports#5]
```
project "/plugins/EC-AuditReports/plugin",{
procedure 'Audit Reports', {
		step 'Approval', {
			subprocedure = 'generateApprovalAuditReport'
			subproject = '/plugins/EC-AuditReports/project'
		}
		step 'Evidence', {
			subprocedure = 'generateEvidenceLinksAuditReport'
			subproject = '/plugins/EC-AuditReports/project'
		}
		step 'Duration', {
			subprocedure = 'generateTaskDurationAuditReport'
			subproject = '/plugins/EC-AuditReports/project'
		}
	}
}
```

**Comment by gmaxey** [comment]
Addressed in v1.4.2

**Comment by gmaxey** [comment]
Addressed in v1.4.2
